db/seeds.rb

```diff
@@ -11,7 +11,7 @@
 Species.where(name: 'Nubian ibex').first_or_create
 Species.where(name: 'Scimitar-horned Oryx').first_or_create
 Species.where(name: 'Sable Antelope').first_or_create
-Species.where(name: 'Tajik Markhor').first_or_create
+Species.where(name: 'Tadjik Markhor').first_or_create
```

Comment:
I usually go for single quotes unless I have variables in there. The only thing I really don't like is something like `'foo' + x + 'bar'`